UI/Source/CollectionViews/mac/CollectionViewController.swift

```diff
@@ -599,7 +599,28 @@ private final class FullWidthScrollView: NSScrollView {
     // MARK: NSScrollView
 
     fileprivate override func scrollWheel(with theEvent: NSEvent) {
-        if scrollEnabled {
+        guard scrollEnabled else {
+            enclosingScrollView?.scrollWheel(with: theEvent)
+            return
+        }
+
+        var selfShouldScroll = true
+        if let enclosingScrollView = enclosingScrollView, let documentView = documentView {
+            let documentVisibleRect = contentView.documentVisibleRect
+            let scrolledPastTop = documentVisibleRect.minY < documentView.frame.minY - contentInsets.top
+            let scrolledPastBottom = documentVisibleRect.maxY > documentView.frame.maxY + contentInsets.bottom
+
+            if scrolledPastBottom && theEvent.scrollingDeltaY < 0.0 {
```

Comment:
Why `<` vs `<=`?

Reply:
In this case, if scrolling delta was 0.0 it would mean the mouse wasn't moving. However in the case above, by making it < and > we allow a little bit of rubber banding to still occur on the inner ScrollView, which feels a little nicer to me instead of having it stop dead.

Reply:
I'm having some trouble whipping up a quick sample project to test, I can try again later, but if you have one handy I'm wondering if you've tried it out with the inner scroll view at rest flush to the top/bottom and starting a scroll from there in the over-scroll direction. It seems to me like the rubber banding might be strange in that case.

Reply:
I actually have it setup like that right now testing a different issue. I'll report back.